Files/Filesystem/Cloud/Providers/GoogleDriveCloudProvider.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Microsoft.Data.Sqlite;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Windows.Storage;
+
+namespace Files.Filesystem.Cloud.Providers
+{
+    public class GoogleDriveCloudProvider : ICloudProviderDetector
+    {
+        public async Task DetectAsync(List<CloudProvider> cloudProviders)
+        {
+            try
+            {
+                // Google Drive's sync database can be in a couple different locations. Go find it. 
+                string appDataPath = UserDataPaths.GetDefault().LocalAppData;
+                string dbPath = @"Google\Drive\user_default\sync_config.db";
+                var configFile = await StorageFile.GetFileFromPathAsync(Path.Combine(appDataPath, dbPath));
+                await configFile.CopyAsync(ApplicationData.Current.TemporaryFolder, "google_drive.db", NameCollisionOption.ReplaceExisting);
+                var syncDbPath = Path.Combine(ApplicationData.Current.TemporaryFolder.Path, "google_drive.db");
+
+                // Build the connection and sql command
+                SQLitePCL.Batteries_V2.Init();
+                using (var con = new SqliteConnection($"Data Source='{syncDbPath}'"))
+                using (var cmd = new SqliteCommand("select * from data where entry_key='root_config__0'", con)) //local_sync_root_path
```

Comment:
I changed "local_sync_root_path" to "root_config__0", because it didn't find Google Drive sync folder, now it is working for me. @gave92 If you use Google Drive sync, can you check does any of this keys exists in your sync_config.db? Another problem here, is when in google drive sync you select two different folders, Files shows only second folder.
![image](https://user-images.githubusercontent.com/20501502/99092378-bb927280-25d9-11eb-92bf-ab72d0b12ea2.png)

Reply:
Hi @tsvietOK yeah I tested with google drive and or me it works using "local_sync_root_path"
I'll check with "root_config__0" as well
This is what I'm using:
![image](https://user-images.githubusercontent.com/9673091/99118470-de338400-25f7-11eb-8f53-72fb988174c5.png)
I don't this I even have the option to synchronize more than one folder

Reply:
I use the same version. You can open Drive sync settings and select other folders to synchronize them.

Reply:
This is my db, I can confirm that "root_config__0" works as well
![image](https://user-images.githubusercontent.com/9673091/99119169-1b4c4600-25f9-11eb-98a2-4fca3ff227b9.png)

Reply:
I just realized you are syncing folders from your pc to google drive. I was syncing stuff I had on my drive to my pc instead.
I think the first uses "root_config__0" the second "local_sync_root_path"

Reply:
About Mega, where did you get an algorithm of sync folder decryption?

Reply:
Did it myself, MEGA is [open source](https://github.com/meganz/MEGAsync) which helped :)